docs/recipes/DevelopingOnFork.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 6
+title: Working with a Forked Network
+description: Learn how to develop and test your dApp using a fork of mainnet network with custom wallets.
+---
+
+# Working with a Forked Network
+This recipe shows how to run a local fork of mainnet and interact with it using Scaffold Stark's contract hooks.
+
+<details open>
+<summary>Here is the complete code for interacting with a forked network:</summary>
```

Comment:
I think we dont need this section as the interaction code doesnt change with forks, and we can try to keep this file cleaner.